File: Cute-T Stickers/Cute-T Stickers/Gmt/FirstViewController.m
//
//  FirstViewController.m
//  FunnyBirthCard
//
//  Created by Ankit Garg on 5/5/20.
//  Copyright © 2020 Yusri. All rights reserved.
//

#import "FirstViewController.h"
#import "ImagesViewController.h"
#import "Cute_T_Stickers-Swift.h"

@interface FirstViewController ()

@end

@implementation FirstViewController

-(void)viewDidLoad
{
    [super viewDidLoad];
}

-(IBAction)useStickers
{
    HomeViewController *vc=[[UIStoryboard storyboardWithName:@"Main" bundle:nil] instantiateViewControllerWithIdentifier:@"HomeViewController"];
    [self.navigationController pushViewController:vc animated:YES];
}

-(IBAction)clickGetLikes
{
    ImagesViewController *VC=[[ImagesViewController alloc] initWithNibName:@"ImagesViewController" bundle:nil];
    
    UINavigationController *navigationController=[[UINavigationController alloc] initWithRootViewController:VC];
    navigationController.modalPresentationStyle=UIModalPresentationFullScreen;
    [navigationController.navigationBar setTranslucent:NO];
    
    [self.navigationController presentViewController:navigationController animated:YES completion:nil];
}

@end

File: Cute-T Stickers/Cute-T Stickers/Gmt/FirstViewController.swift
//
//  FirstViewController.swift
//  Cute-T Stickers
//
//  Created by Ankit Garg on 5/6/20.
//  Copyright © 2020 Omni Soft Solutions. All rights reserved.
//

import UIKit

class FirstViewController: UIViewController
{
    override func viewDidLoad()
    {
        super.viewDidLoad()
    }
    
    @IBAction func useStickers()
    {
        let vc=UIStoryboard(name:"Main", bundle:nil).instantiateViewController(withIdentifier:"HomeViewController")
        navigationController!.pushViewController(vc, animated:true)
    }
    
    @IBAction func clickGetLikes()
    {
        let VC=ImagesViewController(nibName:"ImagesViewController", bundle:nil)
        
        let navigationController=UINavigationController(rootViewController:VC)
        navigationController.modalPresentationStyle = .fullScreen
        navigationController.navigationBar.isTranslucent=false
        
        self.navigationController!.present(navigationController, animated:false, completion:nil)
    }
}
